integration/oasis/chargingstrategy/charge_strategy_creator.go

```diff
@@ -1,14 +1,22 @@
 package chargingstrategy
 
-// ChargingStrategy contains charging related information
-type ChargingStrategy struct {
-	ChargingTime   float64
+// ChargingStatus contains charging related information
+type ChargingStatus struct {
 	ChargingEnergy float64
 }
 
+// ChargingCost represents the cost needed to reach certain states
+type ChargingCost struct {
```

Comment:
Maybe `ChargingStatus`->`State` and `CharingCost`->`Cost` will be better? See https://github.com/golang/go/wiki/CodeReviewComments#package-names

Reply:
Will do it in future pull: https://github.com/Telenav/osrm-backend/issues/211